code-of-conduct.md

```diff
@@ -1,5 +0,0 @@
-## Kubernetes Community Code of Conduct
```

Comment:
because we don't link to it from the main README anymore. We point directly to the CNCF code of conduct in the CNCF repo.

This was agreed in https://github.com/kubernetes/kubernetes/pull/36817